### DISCUSSION TITLE
Upgrading old organizations for HSTS headers

### DISCUSSION THREAD

**Discussion by cszukala** [community/community#10624] [Other Feature Feedback, Questions, & Ideas]
Hey Github, 

A big security feature thats stopping many is:

**- HSTS header (and possibly other related security headers).** 

It would make a incredible difference in security outlook and the confidence of Information Security
people to support github over others.

Thanks :).

**Comment by AndrewKvalheim** [comment]
Looks like #4444 addresses this.
